projects/igniteui-angular/src/lib/combo/combo.component.spec.ts

```diff
@@ -2338,1188 +1031,1591 @@ describe('igxCombo', () => {
+            }));
+            it('should properly call dropdown navigateNext with virtual items', (async () => {
+                expect(combo).toBeDefined();
+                expect(dropdown).toBeDefined();
+                expect(dropdown.focusedItem).toBeFalsy();
+                expect((combo as any).virtDir).toBeDefined();
+                const mockClick = jasmine.createSpyObj('event', ['preventDefault', 'stopPropagation']);
+                const virtualMockUP = spyOn<any>(dropdown, 'navigatePrev').and.callThrough();
+                const virtualMockDOWN = spyOn<any>(dropdown, 'navigateNext').and.callThrough();
+                expect(dropdown.focusedItem).toEqual(null);
+                expect(combo.collapsed).toBeTruthy();
+                combo.toggle();
+                await wait(30);
+                fixture.detectChanges();
+                expect(combo.collapsed).toBeFalsy();
+                (combo as any).virtDir.scrollTo(51);
+                await wait(30);
+                fixture.detectChanges();
+                let items = fixture.debugElement.queryAll(By.css(`.${CSS_CLASS_DROPDOWNLISTITEM}`));
+                let lastItem = items[items.length - 1].componentInstance;
+                expect(lastItem).toBeDefined();
+                lastItem.clicked(mockClick);
+                await wait(30);
+                fixture.detectChanges();
+                expect(dropdown.focusedItem).toEqual(lastItem);
+                dropdown.navigateItem(-1);
+                await wait(30);
+                fixture.detectChanges();
+                expect(virtualMockDOWN).toHaveBeenCalledTimes(0);
+                lastItem.clicked(mockClick);
+                await wait(30);
+                fixture.detectChanges();
+                expect(dropdown.focusedItem).toEqual(lastItem);
+                dropdown.navigateNext();
+                await wait(30);
+                fixture.detectChanges();
+                expect(virtualMockDOWN).toHaveBeenCalledTimes(1);
+                combo.searchValue = 'New';
+                combo.handleInputChange();
+                fixture.detectChanges();
+                await wait(30);
+                items = fixture.debugElement.queryAll(By.css(`.${CSS_CLASS_DROPDOWNLISTITEM}`));
+                lastItem = items[items.length - 1].componentInstance;
+                (lastItem as IgxComboAddItemComponent).clicked(mockClick);
+                await wait(30);
+                fixture.detectChanges();
+                // After `Add Item` is clicked, the input is focused and the item is added to the list
+                // expect(dropdown.focusedItem).toEqual(null);
+                expect(document.activeElement).toEqual(combo.searchInput.nativeElement);
+                expect(combo.customValueFlag).toBeFalsy();
+                expect(combo.searchInput.nativeElement.value).toBeTruthy();
+
+                // TEST move from first item
+                (combo as any).virtDir.scrollTo(0);
```

Comment:
Similar to previous comment you can use combo.virtualScrollContainer.scrollTo instead of (combo as any).virtDir.scrollTo